components/ProjectCards/index.js

```diff
@@ -3,14 +3,22 @@ import OverviewCard from "../OverviewCard";
 
 export default function ProjectCards({ projects }) {
   return (
-    <div>
+    <StyledProjectCards>
       {projects.length === 0 ? (
         <p>no project added to the page</p>
       ) : (
         projects?.map((project) => {
           return <OverviewCard project={project} key={project.id} />;
         })
       )}
-    </div>
+    </StyledProjectCards>
   );
 }
+const StyledProjectCards = styled.div`
```

Comment:
I think an `article` would be a better semantic HTML element here?

Reply:
thx...done it